homeassistant/components/abode/switch.py

```diff
@@ -12,11 +17,15 @@
 ICON = "mdi:robot"
 
 
-async def async_setup_entry(hass, config_entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
     """Set up Abode switch devices."""
     data = hass.data[DOMAIN]
 
-    entities = []
+    entities: list[SwitchEntity] = []
```

Comment:
This type hint is required to take into account `AbodeSwitch` and `AbodeAutomationSwitch`